go.mod

```diff
@@ -121,7 +121,7 @@ require (
 	gopkg.in/mail.v2 v2.3.1
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1
-	xorm.io/builder v0.3.6 // indirect
+	xorm.io/builder v0.3.6
```

Comment:
Nit. Why did this change? Does running `go mod tidy` make this go away?

Reply:
Looks like `main` needs a `go mod tidy` 🤔